examples/export_figure.py

```diff
@@ -99,11 +99,11 @@
 
 # Export figure and change theme before and after exporting to show that the background canvas margins
 # are not in the exported figure.
-viewer.theme = "light"
+viewer.canvas_color = 'white'
 # Optionally for saving the exported figure: viewer.export_figure(path="export_figure.png")
 export_figure = viewer.export_figure()
 scaled_export_figure = viewer.export_figure(scale_factor=5)
-viewer.theme = "dark"
+viewer.canvas_color = 'black'
```

Comment:
```suggestion
viewer.canvas_color = None
```

We want to have theme default color here

Reply:
I thought so too, but my thought was if someone was using light theme, then they will want to have the contrast from screenshot with white to visualization of that on black. I see your perspective, and perhaps someone with light theme would be able to entertain that they would need to switch the first canvas_color to black, for the contrast.